src/utils/values.h

```diff
@@ -117,6 +117,22 @@ int sr_val_build_str_data(sr_val_t *value, sr_val_type_t type, const char *forma
  */
 int sr_dup_val(const sr_val_t *value, sr_val_t **value_dup);
 
+/**
+ * @brief Check if two sr_val_t values are equal.
+ *
+ * @param [in] value1 Sysrepo value to compare
+ * @param [in] value2 Sysrepo value to compare
+ * @return 1 if the values are equal, 0 othewise
+ *
+ * @note If any one of the below parameters of the values differ, the values are not consider equal
+ *  1. type
+ *  2. origin
+ *  3. xpath
+ *  4. default flag
+ *  5. actual value
+ */
```

Comment:
We generally put this additional information in the `brief` description of the function, before the parameters. Even something like
```
Check if two ::sr_val_t values are equal. They are not considered equal if their type, origin, xpath, default flag, or actual value differ.
```
would be fine.